jvm-packages/create_jni.py

```diff
@@ -6,6 +6,7 @@
 import subprocess
 import sys
 from contextlib import contextmanager
+import getopt
```

Comment:
Not used?  Argparser is also quite good.

Reply:
removed for now, we only accept on parameter for testing usage, might be overkill to allow create_jni.py run as commandline tool